package.json

```diff
@@ -12,16 +12,17 @@
     "cache",
     "cacherefresh",
     "assets",
-    "gulp"
+    "gulp",
+    "php"
   ],
   "author": {
-    "name": "Joseph Bisaillon",
+    "name": "Joseph Bisaillon updated by Gabriel Xavier",
     "url": "http://www.josephbisaillon.com"
   },
   "license": "MIT",
   "repository": {
     "type": "git",
-    "url": "https://github.com/josephbisaillon/gulp-cache-refresh.git"
+    "url": "https://github.com/gabrielxavier/gulp-cache-refresh"
```

Comment:
Are you trying to get a new npm package published named, gulp-cache-refresh-php or do you want the php additions to be merged into the current npm package? Just a bit confused on these changes.